pages/gallery.html

```diff
@@ -1,12 +1,60 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="/styles/styles.css">
+	<link rel="stylesheet" href="/styles/gallery.css">
+	<title>Gallery</title>
 </head>
+
 <body>
-  
+	<nav class="nav">
+		<h1>Janice and Ericka's Grey's Anatomy Fan Page!</h1>
+		<h3>Gallery</h3>
+		<a class="active" href="/pages/index.html">Home</a>
+		<a href="/pages/facts.html" rel="import">Fun Facts</a>
+		<a href=/pages/gallery.html rel="import">Gallery</a>
+	</nav>
+	<section class="image-container">
+		<div class="item1">
+			<img src="/images/meredith-grey.jpg" alt="Meredith Grey">
+			<p class="character">Meredith Grey</p>
+		</div>
+		<div class="item2">
+			<img src="/images/cristina_yang.jpg" alt="Cristina Yang">
+			<p class="character">Christina Yang</p>
+		</div>
+		<div class="item3">
+			<img src="/images/alex-karav.jpg" alt="Alex Karev">
+			<p class="character">Alex Karev</p>
+		</div>
+		<div class="item4">
+			<img src="/images/george-omalley.jpg" alt="George O'malley">
+			<p class="character">George O'malley</p>
+		</div>
+		<div class="item5">
+			<img src="/images/izzie-stevens.jpg" alt="Izzy Stevens">
+			<p class="character">Izzie Stevens</p>
+		</div>
+		<div class="item6">
+			<img src="/images/derek-shepard1.jpg" alt="Derek Shepard"> 
+			<p class="character">Derek Shepard</p>
+		</div>
+		<div class="item7">
+			<img src="/images/miranda-bailey.jpg" alt="Miranda Bailey">
+			<p class="character">Miranda Bailey</p>
+		</div>
+		<div class="item8"><img src="/images/richard-webber.jpg" alt="Richard Webber">
+			<p class="character">Richard Webber</p>
+		</div>
```

Comment:
Each of these different divs has unique classes, `item1`, `item2`, ... `item8`.

- If they are unique, I would expect these to be IDs, not classes
- The classes `item5` and up aren't used anywhere, so I would expect these to be deleted